grakn-factory/src/main/java/ai/grakn/factory/TxFactoryJanusHadoop.java

```diff
@@ -43,15 +44,22 @@
  * @author fppt
  */
 public class TxFactoryJanusHadoop extends TxFactoryAbstract<EmbeddedGraknTx<HadoopGraph>, HadoopGraph> {
-    private static final String CLUSTER_KEYSPACE = "janusmr.ioformat.conf.storage.cassandra.keyspace";
-    private static final String INPUT_KEYSPACE = "cassandra.input.keyspace";
+    private static final String JANUSMR_IOFORMAT_CONF_STORAGE_CASSANDRA_KEYSPACE = "janusmr.ioformat.conf.storage.cassandra.keyspace";
```

Comment:
yes, yes. various configuration that essentially meant the same thing. don't ask me why.